x/mongo/driver/operation/abort_transaction.go

```diff
@@ -187,3 +189,13 @@ func (at *AbortTransaction) Retry(retry driver.RetryMode) *AbortTransaction {
 	at.retry = &retry
 	return at
 }
+
+// ServerAPI sets the server API version for this operation.
+func (at *AbortTransaction) ServerAPI(serverAPI *driver.ServerAPIOptions) *AbortTransaction {
+	if at == nil {
+		at = new(AbortTransaction)
+	}
```

Comment:
Does it make sense to factor this common check out into a function like `ensureNotNil`?

Note that actually doing that refactor would probably be outside the scope of this PR.

Reply:
We could do this here and in every operation setter; would save two lines quite often. It would probably end up being `at = ensureNotNil(at)`, which is not quite as clear as this simple if-check in my opinion.